lib/CL/devices/devices.c

```diff
@@ -109,13 +112,16 @@ static init_device_ops pocl_devices_init_ops[] = {
 
 #define POCL_NUM_DEVICE_TYPES (sizeof(pocl_devices_init_ops) / sizeof((pocl_devices_init_ops)[0]))
 
-char pocl_device_types[POCL_NUM_DEVICE_TYPES][30] = {
+char pocl_device_types[POCL_NUM_DEVICE_TYPES][33] = {
```

Comment:
why 33?

Reply:
The size has been increased by three because now three more chars ("tbb") can be added.